Dmitry_Khomich/08.Ansible.Spinup/group_vars/all/all.yaml

```diff
@@ -0,0 +1,4 @@
+ansible_ssh_user: root
+ansible_ssh_pass: QwertY_13
+ansible_ssh_bridge_pass: 23mnZujif4rzU
```

Comment:
You need to declare user/pass just once, do not repeat same, use variables instead, as one of solution